doc/ref_model/chapters/chapter02.md

```diff
@@ -263,19 +265,54 @@ By trying to sort workloads into different categories based on the requirements
     - High network throughput
     - Low network latency
 
-<a name="2.4"></a>
-## 2.4 Cloud Infrastructure Profiles
+## 2.4 Node Profiles & Workload Profiles
 
-Based on the above analysis, following cloud infrastructure profiles are proposed (also shown in **Figure 2-1** below)
+**Node Profiles** are used to tag infrastructure (such as hypervisor hosts, or Kubernetes worker nodes) and associate it with a set of capabilities that are exploitable by the workloads.
+
+**Workload Profiles** are requirements expressed as workload metadata, indicating what kind of infrastructure they must run on to achieve functionality and/or the intended level of performance. They are a resource requesting mechanism, identifying a set of sizing metadata and infrastructure characteristics (such as NUMA alignment, CPU pinning) that are required for the workload to run as intended, then mapped to node flavours at instantiation time.
+A resource request by a workload can be met by any infrastructure node that has the same or a more specialised profile and set of flavours, and the necessary capacity.
+
+Two profile *layers* are proposed:
+
+- The top level **profiles** represent macro-characteristics that partition infrastructure into separate pools, i.e.: an infrastructure object can belong to one and only one profile, and workloads can only be created using a single profile. Workloads requesting a given profile **must** be instantiated on infrastructure of that same profile.
+- For a given profile, **flavours** represent small deviations from (or further qualification, such as sizing) the profile that do not require partitioning the infrastructure into separate pools, but that have specifications with a finer granularity of the profile. Flavours can be *optionally* requested by workloads that want a more granular control over what infrastructure they can run on, i.e.: an infrastructure resource can have **more than one flavour** attached to it, and workloads can request VMs to be instantiated on infrastructure with a certain flavour. Workloads requesting a given flavour **must** be instantiated on infrastructure with that same flavour. It is allowed to instantiate workloads on infrastructure with more flavours than what is requested, as long as the minimum requirements are satisfied.
```

Comment:
We shouldn't call a flavour a deviation from a profile. Maybe we can say "variant" or "characteristic" or additional feature of the node or requirement of a workload.